### ISSUE TITLE
soulseekqt: cannot enter characters in any textbox

### ISSUE THREAD

**Issue by Thunderbottom** [NixOS/nixpkgs#44419]
## Issue description

SoulseekQT does not allow typing in text in any of the textboxes. Running `soulseekqt` from the command line shows the following error:

```
xkbcommon: ERROR: failed to add default include path /usr/share/X11/xkb
Qt: Failed to create XKB context!
Use QT_XKB_CONFIG_ROOT environmental variable to provide an additional search path, add ':' as separator to provide several search paths and/or make sure that XKB configuration data directory contains recent enough contents, to update please see http://cgit.freedesktop.org/xkeyboard-config/ .
```
The application still continues to execute, but no characters can be entered in any textbox.

### Steps to reproduce

1. Install soulseekqt
```sh
    $ nix-env -i soulseekqt
```

2. Run soulseekqt from command line
```sh
    $ soulseekqt
```

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.54, NixOS, 18.03.132847.aec217852f2 (Impala)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.0.4`
 - channels(smolboye): `"nixpkgs"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`

**Comment by xeji** [comment]
cc maintainer @bignaux

**Comment by bignaux** [comment]
update to 2018-1-30, it fix the issue.

**Comment by xeji** [comment]
thank you for fixing this

**Comment by xeji** [comment]
fixed in #44460.

**Comment by Thunderbottom** [comment]
Thanks a lot!
